borders.go

```diff
@@ -374,6 +369,19 @@ func (s Style) applyBorder(str string) string {
 	return out.String()
 }
 
+func (s Style) autoEnableBorder() bool {
```

Comment:
Might call this something like `shouldAutoEnableBorder` just to make it obvious it's a check and not changing state, i.e. actually enabling borders

Reply:
will do

Reply:
Pushed.